### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ishikabansal04**
For #50

### CODE REVIEW

**Comment by soumyaa1804** [comment]
@ishikabansal04, yes you updated what all I recommended for the code but as I said earlier these changes are to be made on the ishikabansal04.md file that you created in task 2 and not on the README.md file. Kindly cut and paste the content you added to the file ishikabansal04.md which would be in the `participants` folder.

**Comment by soumyaa1804** [comment]
And you need not to make a different branch to do this just add commits to this branch only and push it.

**Comment by soumyaa1804** [comment]
@ishikabansal04 did you created a new branch with the same naming conventions as of task2 branch after deleting the task2 branch? If yes, then it is fine. But if you are working on a previous merged branch it may be a problem. Let us know if you have created a new branch `ishikabansal04-task2` after deleting the one associated with the previous task.

**Comment by ishikabansal04** [comment]
I deleted the prev branch(using git bash) and created a new branch styling-md-ishikabansal (as per instructions given in task 3)
but i am in doubt if my prev branch was deleted because my git bash shows only 2 branches currently (master and styling-md-ishikabansal) but when i click on branch option on my github directly it shows 3 branches including ishikabansal04-task2.

File: README.md [hunk 1]
```diff
@@ -30,3 +30,12 @@ For participating in this contest, register [here](https://bit.ly/3c3lNQY) and g
...
 
+### My_Experience
+I am loving solving these daily tasks and really learning a lot from these.
+
+Thankyou *Acm Igdtuw* for organsing this event. 
+
+
+1. Task 1 made me learn some basics of github which I was not clear about initially.
+2. In task 2 I learned how to create a PR.
+3. Here in task 3 I am learning some markdown file editing features.
```
Comment (soumyaa1804) [inline README.md:41]: You need to remove these changes as well.

File: participants/ikabansal04.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
-Name: Ishika Bansal
-Github Username: ikabansal04
-University: Indira Gandhi Delhi Technical University for Women
-Year: 2
-Branch: CSE
+Name: Ishika Bansal     
+Github Username: ikabansal04   
+University: Indira Gandhi Delhi Technical University for Women    
+Year: 2       
+Branch: CSE
```
Comment (soumyaa1804) [inline participants/ikabansal04.md:5]: Giving line breaks would be really a good idea. Leaving a line or enter twice to give a line break.

**Comment by soumyaa1804** [comment]
> but when i click on branch option on my github directly it shows 3 branches including ishikabansal04-task2.

Reading [this](https://www.freecodecamp.org/news/how-to-delete-a-git-branch-both-locally-and-remotely/) would answer your question. :)

File: README.md (hunk 1 above)
Comment (ishikabansal04) [reply README.md:41]: Please check once . I have removed all the changes from README.md file and made the changes in my task2 file.Please review my changes once

**Comment by ishikabansal04** [comment]
I have deleted my prev branch remotely as well .Please tell me if it is correct or still has some problem. @soumyaa1804

**Pull request closed without merging**